### ISSUE TITLE
Question: Two Previews or Two Separate Context Menus

### ISSUE THREAD

**Issue by ChristopherGabba** [nandorojo/zeego#89]
Good morning,
I'm trying to build a "dual" context menu of sorts and I have two options:
- Press and hold on an image and one menu opens below and a seperate context menu opens off to the right, or
- Press and hold on an image and one menu opens below and a separate view opens off ot he right of the image (i.e. a second `ContextMenu.Preview`)

I'm struggling to get this working. How could I have two separate menus on one preview?

**Comment by nandorojo** [comment]
two context menus is not supported nor allowed by native APIs. Consider use a Group + Label if you want to. Or nested menus.

Another (iOS-only) option is to use the Auxiliary view to render an arbitrary view, however it won’t work for a full context menu. But perhaps you have things like message reactions or something that could go there?

**Comment by ChristopherGabba** [comment]
@nandorojo Yeah the Auxilliary view would work if it didn't act that finicky. The iOS message emoji is similar to what I'm trying to accomplish (i.e. separate options)

![IMG_89C99F35F084-1](https://github.com/nandorojo/zeego/assets/122328438/a626577f-6277-4c85-b9c5-717f5b0da181)

Here is the exact goal, but I could get away with turning the text at the top into a separate context menu as well. Sounds like that is not possible:
![IMG_89C99F35F084-1](https://github.com/nandorojo/zeego/assets/122328438/f75ab4c5-1fe1-4577-acb8-e5517ee529e9)

**Comment by ChristopherGabba** [comment]
@nandorojo The Auxiliary view actually is doing exactly what I want it to. I didn't see this thing in the Docs anywhere. I do agree though it's a little finicky but appears to be working well for the most part...

**Comment by andreasbroch2** [comment]
This was exactly what i was looking for. I don't know if you don't think it's ready, but I would suggest adding it to the docs. 

I was trying to modify the preview to do both reactions and preview which is not easy, before finding this. This works so much better! 

Thank you for this great tool!

**Comment by nandorojo** [comment]
Oh, yeah I should add that to the docs. Anyone down to PR?
